lib/chef/provisioning/google_driver/driver.rb

```diff
@@ -151,12 +150,12 @@ def stop_machine(action_handler, machine_spec, machine_options)
       name = machine_spec.name
       instance = instance_for(machine_spec)
 
-      if instance.nil? || instance[:status] == "TERMINATED"
+      if instance.nil? || instance.terminated?
         raise "Machine #{name} does not have an instance associated with it, or instance does not exist."
       end
 
-      if instance[:status] != "STOPPED"
-        if instance[:status] != "STOPPING"
+      if !instance[:status].stopped?
```

Comment:
I wold change this to `unless` block since it's a negative statement and there's no else:
```
unless instance[:status].stopped?
```